common/api-review/app.api.md

```diff
@@ -71,6 +71,20 @@ export interface FirebaseOptions {
     storageBucket?: string;
 }
 
+// @public
+export interface FirebaseServerApp extends FirebaseApp {
+    authIdTokenVerified: () => Promise<void>;
+    name: string;
+    readonly settings: FirebaseServerAppSettings;
```

Comment:
can we change this to be a getter, so it won't serialize to json by default?

Reply:
Sure thing! Will work on that today.

Reply:
Looks like [it already has a getter,](https://github.com/firebase/firebase-js-sdk/blob/7ba816a607504f363878a6401c23a9b0a74618a5/packages/app/src/firebaseServerApp.ts#L113) akin to  [FirebaseOptions in FirebaseApp](https://github.com/firebase/firebase-js-sdk/blob/7ba816a607504f363878a6401c23a9b0a74618a5/packages/app/src/firebaseApp.ts#L75), but let me know if something looks out of place.

Reply:
should we update the interface to match, `get` vs `readonly`?

Reply:
I'm following the same convention of `FirebaseOptions` in `FirebaseApp` which defines it as `readonly`.